### PR TITLE
Language versions

### PR DESCRIPTION
**Pull request by hansvancalster**
This PR is related to discussion in https://github.com/inbo/protocolhelper/pull/35.

### CODE REVIEW

**Review by ThierryO — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,8 +46,8 @@ The workflow is as follows for a **new** protocol:
 
 1. make sure your local clone of the remote repository is up to date:
     1. with the master branch checked out, press the pull button in the Git pane
-1. a subject-matter specialist uses `protocolhelper::create_sfp()` to start a new [protocol from a template](#from-a-new-template)
-1. the generated protocol-code (e.g. sfp-406) is noted and a new branch named after the protocol-code is created:
+1. a subject-matter specialist uses `protocolhelper::create_protocol()` to start a new [protocol from a template](#from-a-new-template)
```
Comment (florisvdh) [inline CONTRIBUTING.md:49]:
```suggestion
1. a subject-matter specialist uses `protocolhelper::create_protocol()` (or one of its shortcut functions `create_sfp()` or `create_spp()`) to start a new [protocol from a template](#from-a-new-template)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -80,13 +80,13 @@ The workflow is as follows for a **new** protocol:
...
 
-- you don't need `protocolhelper::create_sfp()`
+- you don't need `protocolhelper::create_protocol()`
 - the creation of the new branch can (re-)use the protocol-code of the existing protocol
 - after review is finished, the protocol-specific `NEWS.Rmd` should be updated to document the substantive changes between the updated version of the previous version.
 
 For adding a **pre-existing version of a protocol that was written in `docx` format**, follow the steps to create a new protocol, except in the second step:
 
-- a subject-matter specialist uses `protocolhelper::create_sfp()` to convert the `docx` protocol to Rmarkdown files. See section [From an existing docx protocol](#from-an-existing-docx-protocol).
+- a subject-matter specialist uses `protocolhelper::create_protocol()` to convert the `docx` protocol to Rmarkdown files. See section [From an existing docx protocol](#from-an-existing-docx-protocol).
```
Comment (florisvdh) [inline CONTRIBUTING.md:89]:
```suggestion
- a subject-matter specialist uses `protocolhelper::create_protocol()` (or one of its shortcut functions `create_sfp()` or `create_spp()`) to convert the `docx` protocol to Rmarkdown files. See section [From an existing docx protocol](#from-an-existing-docx-protocol).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -143,18 +142,34 @@ create_sfp(title = "titel van het protocol",
...
+           date = Sys.Date(), 
+           reviewers = c("reviewer1, reviewer2"), 
+           file_manager = "manager",
+           project_name = "mne",
+           language = "nl",
+           version_number = "2021.01",
+           render = FALSE)
+```
+
+For project-specific protocols, `protocolhelper::add_subprotocol()` can be used subsequently to add (sections of) published standard field protocols as subprotocols.
```
Comment (florisvdh) [inline CONTRIBUTING.md:162]: Best add that a specific version needs to be set for each subprotocol (it should not evolve dynamically within a project protocol version). Optionally, project-specific parameters can be passed.

File: README.md [hunk 4]
```diff
@@ -77,34 +77,39 @@ The incremental nature means that general tags reflect the time sequence of prot
...
-The `##` indicates an incremental number. 
-For instance, the first field protocol for "theme water" will have protocol code `sfp-101`.
 The `s` and `p` refer to **s**tandard **p**rotocol, while `f, i, o, a, p` indicate the first letter of the protocol type.
 
-The last three digits of the protocol-code will be referred to as the **protocol-number**. 
+The three digits of the protocol-code will be referred to as the **protocol-number**.
 In case of thematic protocols, the first digit of the protocol-number corresponds with the theme-number.
+The `##` indicates an incremental number. 
+For instance, the first field protocol for "theme water" will have protocol code `sfp-101-nl`.
+The `s*p-###` part of the protocol can be thought of as a code that corresponds one to one with the title of the protocol.
```
Comment (florisvdh) [inline README.md:101]: > corresponds one to one with the title of the protocol

Titles are different between languages.

```suggestion
The `s*p-###` part of the protocol can be thought of as a code that corresponds one-to-one with the title of the protocol (when ignoring language).
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -143,18 +142,34 @@ create_sfp(title = "titel van het protocol",
...
+
+## Protocol language
 
 Protocols can be written in either one or multiple languages. 
 The functions in the `protocolhelper` package support Dutch (language = 'nl') and English (language = 'en').
+A translation of the same protocol will be evident from the protocol-code, e.g.: `sfp-001-nl` and `sfp-001-en`.
```
Comment (florisvdh) [inline CONTRIBUTING.md:168]:
```suggestion
A translation of the same protocol will be evident from the protocol-code, e.g.: `sfp-001-nl` and `sfp-001-en`.
Note that these must be considered as just two different protocols (with a different protocol code), although the protocol prefix+number shows their relationship.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -143,18 +142,34 @@ create_sfp(title = "titel van het protocol",
...
+To indicate that `sfp-001-en` is a literal translation of `sfp-001-nl`, the protocol-specific `NEWS.Rmd` file should mention this together with the version number of `sfp-001-nl`.
```
Comment (florisvdh) [inline CONTRIBUTING.md:169]:
```suggestion
To indicate that a specific version of `sfp-001-en` is a literal translation of a version of `sfp-001-nl`, the `NEWS.Rmd` file of `sfp-001-en` should mention this, in a version-specific way.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -143,18 +142,34 @@ create_sfp(title = "titel van het protocol",
...
+Different language versions of the same protocol should be added to the repo in a separate pull request.
```
Comment (florisvdh) [inline CONTRIBUTING.md:170]:
```suggestion
Different language variants of the 'same' protocol should be added to the repo in a separate pull request.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -143,18 +142,34 @@ create_sfp(title = "titel van het protocol",
...
+Note that in subsequent versions of - say - `sfp-001-en` it is allowed that it's contents diverges from the version of `sfp-001-nl` of which it was a literal translation.
```
Comment (florisvdh) [inline CONTRIBUTING.md:171]:
```suggestion
Note that in subsequent versions of - say - `sfp-001-en` it is allowed that its contents diverge from the version of `sfp-001-nl` of which it was a literal translation.
```

File: README.md [hunk 9]
```diff
@@ -77,34 +77,39 @@ The incremental nature means that general tags reflect the time sequence of prot
 
 ### Protocol-code and protocol-number <a name="protocol-code"></a>
 
-A **protocol-code** consists of three characters and three digits separated by a hyphen.
+A **protocol-code** consists of three characters, three digits and two characters separated by a hyphen.
```
Comment (florisvdh) [inline README.md:80]:
```suggestion
A **protocol-code** consists of a prefix (three characters), a protocol-number (three digits) and a language tag (two characters), all separated by a hyphen.
```

**Review by florisvdh — APPROVED** [review]
Well documented @hansvancalster. :+1:  Some minor changes suggested for further clarification.

**Pull request merged**